### PR TITLE
Solving bug of missing labels

### PR DESCRIPTION
**Pull request by NickNtamp**
With this PR we are adding the labels as a parameter in evaluation pipelines. Also I added an edge case unit test per pipeline. This solves the issue but there are some open questions and pending work:

- @sinnec can you please connect this with the pipeline by feeding the possible labels, as we discussed. Don't forget that we are asking for the values of client's input.
- @momegas we observed along with @sinnec that the labels are optional in models.py under schema folder. This should be mandatory. Also there is a question here, what will happen if the client insert the labels in wrong way? Maybe it is better in our next iteration to find the labels by our own and don't asking them by the client at all. It is pretty easy as we just seeking for the unique values of the target column of training set.

### CODE REVIEW

**Review by momegas — APPROVED** [review]

File: src/cron_tasks/monitoring_metrics.py [hunk 1]
```diff
@@ -82,11 +82,13 @@ async def run_calculate_performance_metrics_pipeline(
         )
         return
 
+    labels = list(model.labels.values())
+
```
Comment (sinnec) [inline src/cron_tasks/monitoring_metrics.py:86]: This solution is going to be OK if those labels are given by the user. But what if they aren't provided since they're optional? What do you think @momegas ?

Comment (NickNtamp) [reply src/cron_tasks/monitoring_metrics.py:86]: Asked the same above in a different way @sinnec.....
